src/main/java/com/rabbitmq/client/impl/recovery/AutorecoveringChannel.java

```diff
@@ -67,21 +67,26 @@ public void close() throws IOException, TimeoutException {
         try {
             delegate.close();
         } finally {
-            for (String consumerTag : consumerTags) {
-                this.connection.deleteRecordedConsumer(consumerTag);
-            }
-            this.connection.unregisterChannel(this);
+            recoveryCleanup();
```

Comment:
Is this change really related to delay calculation and back-off?

Reply:
no, just noticed the recorded consumers weren't being deleted with the 2nd close method like the 1st.

Reply:
removed for now. could be a separate future PR